src/main/java/uk/gov/hmcts/reform/civil/handler/tasks/PollingEventEmitterHandler.java

```diff
@@ -15,7 +15,7 @@
 @Slf4j
 @RequiredArgsConstructor
 @Component
-@ConditionalOnExpression("${POLLING_EVENT_EMITTER_ENABLED:true}")
+@ConditionalOnExpression("${polling.event.emitter.enabled:true}")
```

Comment:
@akriti-hmcts 

The PollingEventEmitter is already disapled in Low Level Envs:
https://github.com/hmcts/civil-service/blob/master/charts/civil-service/values.aat.template.yaml#L65
https://github.com/hmcts/civil-service/blob/master/charts/civil-service/values.preview.template.yaml#L65

Reply:
@HarryH96 its misconfigured and thats why it is getting invoked in demo